pkg/virt-handler/node-labeller/api/capabilities.go

```diff
@@ -107,8 +107,8 @@ func (b *yesnobool) UnmarshalXMLAttr(attr xml.Attr) error {
 
 func (b *CPUSiblings) UnmarshalXMLAttr(attr xml.Attr) error {
 	if attr.Value != "" {
-		if list, err := hwutil.ParseCPUSetLine(attr.Value, 100); err == nil {
-			for _, cpu := range list {
+		if cpuSet, err := cpuset.Parse(attr.Value); err == nil {
+			for _, cpu := range cpuSet.List() {
```

Comment:
nit: Is there any particular reason to have the list sorted?

Reply:
Good question :D This is unmarshaing so probably doesn't really make sense. Let me know if I should change to `UnsortedList()`.

Reply:
As per offline discussion, it has been decided to always use `List()`.
Thanks